Uplift/Views/Histogram.swift

```diff
@@ -214,6 +215,19 @@ class Histogram: UIView {
                 selectedIndex = indexSelected
                 selectedBar.backgroundColor = .fitnessSelectedYellow
 
+<<<<<<< HEAD
+        // update selectedTime and the descriptor
+        if data[selectedIndex + openHour] < mediumThreshold {
+            selectedTimeDescriptor.text = timeDescriptors[0]
+        } else if data[selectedIndex + openHour] < highThreshold {
+            selectedTimeDescriptor.text = timeDescriptors[1]
+        } else {
+            selectedTimeDescriptor.text = timeDescriptors[2]
+        }
+        selectedTimeDescriptor.sizeToFit()
+        
+        selectedTime.text = hours.openTime.addingTimeInterval( Double(selectedIndex) * secondsPerHour ).getStringOfDatetime(format: "ha :")
```

Comment:
make sure you resolve this conflict and remove the <<<<< HEAD and >>>>>> master flags!